src/detection/card/items/camera/item.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Typography } from '@material-ui/core'
+
+import { useStyles } from 'detection/card/items/style'
+import { useItem } from 'detection/card/use-item'
+import { useOnClick } from 'detection/card/items/camera/hooks/use-on-click'
+
+export const Item = () => <>
```

Comment:
Also, `item` is generally a very bad naming choice, since it's very generic and thus it doesn't provide too much context. Let's rename this to something like `camera`. We can also change its folder structure to accommodate this change.

Reply:
Done